hw/top_earlgrey/data/ip/chip_pwrmgr_testplan.hjson

```diff
@@ -82,7 +82,7 @@
             SiVal: No need to run this, run chip_sw_pwrmgr_random_sleep_all_wake_ups instead.
             '''
       stage: V2
-      si_stage: NA
+      si_stage: SV3
```

Comment:
Please notice line 82: there is a more comprehensive test that covers this functionality, and this will only add CI runtime.
If we really intend to run these tests, please remove line 82, here and in the other two pwrmgr tests that were marked as not needed.

Reply:
Removed line 82